clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.kt

```
@@ -0,0 +1,991 @@
+        // check if there is app launched/wzrk_fetch event
+        for (index in 0..<body.queue.length()) {
+            try {
+                val event = body.queue.getJSONObject(index)
+                val eventType = event.getString("type")
+                if ("event" == eventType) {
+                    val evtName = event.getString("evtName")
+                    if (Constants.APP_LAUNCHED_EVENT == evtName || Constants.WZRK_FETCH == evtName) {
+                        return true
+                    }
+                }
+            } catch (jsonException: JSONException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)